File: implementations/node/sdk/src/contracts/publication/publication-dependency-package-kind.ts
export const AI_SDK_PUBLICATION_DEPENDENCY_PACKAGE_KINDS = [
  "PythonWheelBundle",
  "NodeLockedBundle",
  "DotNetAssemblyClosure",
] as const;

export type AiSdkPublicationDependencyPackageKind =
  (typeof AI_SDK_PUBLICATION_DEPENDENCY_PACKAGE_KINDS)[number];

File: implementations/python/sdk/src/multiplexed_ai_sdk/contracts/publication/publication_dependency_package_kind.py
from enum import Enum


class AiSdkPublicationDependencyPackageKind(str, Enum):
    PYTHON_WHEEL_BUNDLE = "PythonWheelBundle"
    NODE_LOCKED_BUNDLE = "NodeLockedBundle"
    DOTNET_ASSEMBLY_CLOSURE = "DotNetAssemblyClosure"
